### PR TITLE
Update README.md information

### PR DESCRIPTION
**Pull request by anaschwendler**
**What is the purpose of this Pull Request?**
The objective of this PR is to update the information of the `README.md` of the project, that was the same from when we started working. Many things have happened after that :)

**What was done to achieve this purpose?**
Everyone in the project worked on this task: reviewing the text, and giving opinions 

**How to test if it really works?**
Just check if the markdown is working

**Who can help reviewing it?**
@Irio @cuducos @jtemporal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,25 +2,22 @@
...
 
 ## Support Serenata de Amor
 
-We [were crowdfunded](https://catarse.me/serenata) to operate for the first two months. We're really glad for this support. However we want to go further — if you wanna make us go further:
+In 2016, precisely on the September 7th, our [first crowdfunding](https://catarse.me/serenata) campaign was launched and we raised approximately R$ 80,000 – 30% more than our initial target. That allowed us to afford the dedicated team for the first 90 days.
+
+At that time, we developed Rosie and made our first joint complaint, with 629 cases reported to the Chamber. Without recurrence, however, we can not continue the work. That is why we have created a space for people who want to contribute frequently, even with minor values, to our project. However we want to go further — if you wanna make us go further:
 
 * Participate in our [recurring crowdfunding campaign on Apoia.se](http://apoia.se/serenata)
 * Donate via Bitcoin:[`1Gbvfjmjvur7qwbwNFdPSNDgx66KSdVB5b`](bitcoin:1Gg9CVZNYmzMTAjGfMg62w3b6MM7D1UAUV?amount=0.01&message=Supporting%20Serenata%20de%20Amor%20Operation)
```
Comment (Irio) [inline README.md]: Can you link the code to Blockchain.info?

```
* Donate via Bitcoin: [1Gbvfjmjvur7qwbwNFdPSNDgx66KSdVB5b](https://blockchain.info/address/1Gbvfjmjvur7qwbwNFdPSNDgx66KSdVB5b)
```

File: README.md [hunk 2]
```diff
@@ -2,25 +2,22 @@
...
+**[Visit our official website](http://serenatadeamor.org/)** and remember to read **[the contribution guide](/CONTRIBUTING.md)** if you plan to help on the software side.
 
 ## Support Serenata de Amor
 
-We [were crowdfunded](https://catarse.me/serenata) to operate for the first two months. We're really glad for this support. However we want to go further — if you wanna make us go further:
+In 2016, precisely on the September 7th, our [first crowdfunding](https://catarse.me/serenata) campaign was launched and we raised approximately R$ 80,000 – 30% more than our initial target. That allowed us to afford the dedicated team for the first 90 days.
+
+At that time, we developed Rosie and made our first joint complaint, with 629 cases reported to the Chamber. Without recurrence, however, we can not continue the work. That is why we have created a space for people who want to contribute frequently, even with minor values, to our project. However we want to go further — if you wanna make us go further:
 
 * Participate in our [recurring crowdfunding campaign on Apoia.se](http://apoia.se/serenata)
```
Comment (Irio) [inline README.md]: *APOIA.se

File: README.md [hunk 3]
```diff
@@ -2,25 +2,22 @@
...
 
-Our current milestone is to create the means for this kind of automation with the Quota for Exercising Parliamentary Activity (CEAP), from the Brazilian Chamber of Deputies. This job includes the development of APIs, data cleaning and analyses, conception and validation of scientific hypothesis, confirmation of illicit acts via investigation and reports - to the population and to legal authorities.
-
-To achieve this goal, unprecedented, we invite everyone to train the intelligence, collect information, cross databases, validate hypothesis and apply Machine Learning with models competing against each other and getting combined in ensembles with higher precision than any previous option.
 
 ## Before contributing
 
-**[Visit our official website](http://serenata.datasciencebr.com/)** and remember to read **[the contribution guide](/CONTRIBUTING.md)** if you plan to help on the software side.
+**[Visit our official website](http://serenatadeamor.org/)** and remember to read **[the contribution guide](/CONTRIBUTING.md)** if you plan to help on the software side.
```
Comment (Irio) [inline README.md]: Can you link to https://serenatadeamor.org/?

**Comment by Irio** [comment]
Ready to merge once you make these changes.

File: README.md (hunk 3 above)
Comment (anaschwendler) [reply README.md]: I'm already linking to it, I guess what you said is  https://serenatadeamor.org/en maybe?

Comment (anaschwendler) [reply README.md]: done.

**Comment by anaschwendler** [comment]
@Irio done here!

**Pull request merged**